tools/tsp-client/package-lock.json

```diff
@@ -9,8 +9,7 @@
       "version": "0.12.2",
```

Comment:
Actually this looks wrong like the package-lock.json wasnt updated correctly. If you do an npm install is should fix all dependencies based on package.json.